app/src/main/java/com/android/sample/model/recipe/MealDBRecipesRepository.kt

```diff
@@ -190,7 +191,7 @@ class MealDBRecipesRepository(private val client: OkHttpClient) : RecipesReposit
   }
 
   override fun listCategories(onSuccess: (List<String>) -> Unit, onFailure: (Exception) -> Unit) {
-    val url = "$MEAL_DB_URL/categories.php"
+    val url = "https://www.themealdb.com/api/json/v1/1/categories.php"
```

Comment:
This can be transformed to this :
```suggestion
   val url = MEAL_DB_URL +"categories.php"
```

Reply:
You're right; I changed it for testing and forgot to reset it to the original.